### PR TITLE
Add Edwin Kaburu.md

### PR DESCRIPTION
**Pull request by EdwinKaburu**
https://raw.githubusercontent.com/EdwinKaburu/EdwinKaburu.github.io/add-EdwinKaburu-profile/pages/vi/profiles/EdwinKaburu

### CODE REVIEW

**Comment by robinGiri** [comment]
@EdwinKaburu Please use at least 5 different types of Markdown elements. And your preview page doesn't seem like [profile 1](https://open-learning-exchange.github.io/#!./pages/vi/profiles/Loshma93.md) or any other example.

**Comment by EdwinKaburu** [comment]
Ok Will do

**Comment by EdwinKaburu** [comment]
Fixes #

Description
Created a markdown file for my profile

Raw.Ediwn preview link
https://raw.githack.com/EdwinKaburu/EdwinKaburu.github.io/add-EdwinKaburu-profile/#!pages/vi/profiles/EdwinKaburu.md

File: pages/vi/profiles/EdwinKaburu.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Hello Everyone
+I am Edwin Kaburu. I am from Kent, WA (TimeZone : Pacific Time). I am currently using Windows 10
+I am currently pursing my Bachelors at Renton Technical College. ![alt text](https://www.rtc.edu/sites/default/files/vital-rtccampus-1000x500.jpg)
+I have a Application Developer Certificate from Renton Technical College. The technical skills that I have come to learn over the past few years include
+ - C#
```
Comment (lmmrssa) [inline pages/vi/profiles/EdwinKaburu.md]: @EdwinKaburu this is not creating bullets, please also check githack link for output

**Comment by singharpita** [comment]
Old PR closing

**Pull request closed without merging**